build/run_unit_group.sh

```diff
@@ -134,7 +134,7 @@ function test_group_other() {
            -Dexclude='**/ManagedLedgerTest.java,
                    **/OffloadersCacheTest.java
                   **/PrimitiveSchemaTest.java,
-                  BlobStoreManagedLedgerOffloaderTest.java'
+                  BlobStoreManagedLedgerOffloaderTest.java' -DtestReuseFork=false
```

Comment:
@codelipenghui do you remember why this was needed? I added #17943 to remove it.

Reply:
The motivation is here https://github.com/apache/pulsar/issues/17152

Although I think it's safe to add it back